CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - use `SOURCE_DATE_EPOCH` environment var for build timestamp instead of `Date.now()` if set.
 - use italic variants of Roboto font correctly #3949
 - show chat name when searching in chat #3950
+- upgrades react to v18 and react pinch pan zoom to v3
```

Comment:
move that to the top, your change is not released yet ;)

Reply:
sowwy